### ISSUE TITLE
E121 - undefined var g:airline

### ISSUE THREAD

**Issue by TheRojam** [mkitt/tabline.vim#6]
```
$ vim .vimrc
error by executing  "/home/therojam/.vimrc":
line  54:
E121: undefomed var g:airline

```
I installed tabline via vundle and wanted to enable it via .vimrc with airline.
See above for the error i get by starting vim

my .vimrc
set nocompatible              " be iMproved, required
filetype off                  " required

" set the runtime path to include Vundle and initialize
set rtp+=~/.vim/bundle/Vundle.vim
call vundle#begin()
" alternatively, pass a path where Vundle should install plugins
"call vundle#begin('~/some/path/here')

" let Vundle manage Vundle, required
Plugin 'VundleVim/Vundle.vim'

" The following are examples of different formats supported.
" Keep Plugin commands between vundle#begin/end.
" plugin on GitHub repo
Plugin 'tpope/vim-fugitive'
" plugin from http://vim-scripts.org/vim/scripts.html
Plugin 'L9'
"tabline plugin
Plugin 'mkitt/tabline.vim'
" tagbar
Plugin 'majutsushi/tagbar'
" synstastic - syntax errors
Plugin 'vim-syntastic/syntastic'
" vim-airline-themes via .vim-folder
Plugin 'vim-airline/vim-airline-themes'

" All of your Plugins must be added before the following line
call vundle#end()            " required
filetype plugin indent on    " required
" To ignore plugin indent changes, instead use:
"filetype plugin on
"
" Brief help
" :PluginList       - lists configured plugins
" :PluginInstall    - installs plugins; append `!` to update or just :PluginUpdate
" :PluginSearch foo - searches for foo; append `!` to refresh local cache
" :PluginClean      - confirms removal of unused plugins; append `!` to auto-approve removal
"
" see :h vundle for more details or wiki for FAQ
" Put your non-Plugin stuff after this line
"
" colored syntax highlighting
syntax on
" set airline statusline
:set laststatus=2
"turn powerline fonts on
let g:airline_powerline_fonts = 1
" set theme at start
" let g:airline_theme='dark'
let g:airline_theme='bubblegum'

let g:airline"extensions"tabline"enabled = 1
let g:airline"extensions"tagbar"enabled = 1
let g:airline"extensions"syntastic"enabled =1

	
" => Backups ausschalten
set nobackup 
" => Zeile und Spalte anzeigen
set ruler                  

" =>  ~/.exrc nicht benutzen
set noexrc                
" => nicht piepsen, wenn man an Textbegrenzungen anstößt o.ä.
set noerrorbells          
" => den aktuellen Mode anzeigen
set showmode              
" => Zeilennummern anzeigen
set nu

**Comment by mkitt** [comment]
@TheRojam I'm sorry not sure what the issue could be, I'm not a vim airline user. You might want to investigate one of the [forks of this repository](https://github.com/mkitt/tabline.vim/network). Someone there might have a working version with airline.

**Comment by chrisbra** [comment]
> let g:airline"extensions"tabline"enabled = 1
> let g:airline"extensions"tagbar"enabled = 1
> let g:airline"extensions"syntastic"enabled =1

That is wrong. The quote starts a comment, so vim only sees the `g:airline` variable. which is unknown, which is the cause for your error.
